sections/shorting/ShortingCard/ShortingCard.tsx

```diff
@@ -11,24 +11,43 @@ import useShort from '../hooks/useShort';
 
 import { CurrencyCardsSelector, ExchangeCardsWithSelector } from 'styles/common';
 
+import { useTranslation, Trans } from 'react-i18next';
+import { useRecoilValue } from 'recoil';
+import { isL2State } from 'store/wallet';
+
+import { MessageContainer, Message } from '../common';
+import NetworksSwitcher from 'sections/shared/Layout/AppLayout/Header/NetworksSwitcher';
+
 const ShortingCard: FC = () => {
 	const { quoteCurrencyCard, baseCurrencyCard, footerCard } = useShort({
 		defaultBaseCurrencyKey: Synths.sETH,
 		defaultQuoteCurrencyKey: Synths.sUSD,
 	});
 
+	const { t } = useTranslation();
+	const isL2 = useRecoilValue(isL2State);
+
 	return (
 		<Container>
-			<ConvertContainer>
-				<ExchangeCardsWithSelector>
-					{quoteCurrencyCard}
-					{baseCurrencyCard}
-					<StyledCurrencyCardsSelector>
-						<CRatioSelector />
-					</StyledCurrencyCardsSelector>
-				</ExchangeCardsWithSelector>
-				<ExchangeFooter>{footerCard}</ExchangeFooter>
-			</ConvertContainer>
+			{isL2 ? (
+				<ConvertContainer>
+					<ExchangeCardsWithSelector>
+						{quoteCurrencyCard}
+						{baseCurrencyCard}
+						<StyledCurrencyCardsSelector>
+							<CRatioSelector />
+						</StyledCurrencyCardsSelector>
+					</ExchangeCardsWithSelector>
+					<ExchangeFooter>{footerCard}</ExchangeFooter>
+				</ConvertContainer>
+			) : (
+				<MessageContainer>
+					<Message>
+						<Trans t={t} i18nKey="shorting.l1-deprecated" />
+					</Message>
+					<NetworksSwitcher />
```

Comment:
Thinking this approach might be better in the event that there's regression in the NetworkSwitcher component.

Also might make more sense to use our default button style (the gold one) since that's typically used for messages in this format.